src/peer-store/index.js

```diff
@@ -217,6 +217,16 @@ class PeerStore extends EventEmitter {
       protocols: Array.from(peerInfo.protocols)
     })
   }
+
+  /**
+   * Returns the known multiaddrs for a given `PeerId`
+   * @param {PeerId} peerId
+   * @returns {Array<Multiaddr>}
+   */
+  multiaddrsForPeer (peerId) {
+    const peerInfo = this.get(peerId.toB58String())
```

Comment:
We really need to move away from `toB58String`!

Reply:
We can do a followup PR to replace `toB58String` with `toString` everywhere. One of the things we started moving towards is just taking `PeerId` in the APIs to avoid the discrepancy. Then internally any PeerId string can be used, as long as it's consistent in that system.